crates/bevy_window/src/window.rs

```diff
@@ -609,8 +623,13 @@ pub struct WindowDescriptor {
     /// macOS X transparent works with winit out of the box, so this issue might be related to: <https://github.com/gfx-rs/wgpu/issues/687>
     /// Windows 11 is related to <https://github.com/rust-windowing/winit/issues/2082>
     pub transparent: bool,
-    #[cfg(target_arch = "wasm32")]
+    /// The "html canvas" element selector. If set, the given selector will be used to find a matching html canvas element,
```

Comment:
Does this actually take an arbitrary CSS selector, or just an id selector? The docstring implies the former, but the examples seem to only select ids.

Reply:
a css selector. it uses https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector

Reply:
I'll add a link to those docs.